core/src/main/java/com/scalar/db/api/ScanBuilder.java

```diff
@@ -1582,83 +1580,4 @@ public BuildableScanFromExistingWithOngoingWhereAnd and(OrConditionSet orConditi
       return this;
     }
   }
-
-  private static class Where {
```

Comment:
Moved to `SelectionBuilder` to commonly use in both `GetBuilder` and `ScanBuilder`.